org.mixedrealitytoolkit.windowsspeech/Tests/Editor/PackageValidationTest.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+#if HAS_ASSET_STORE_VALIDATION
+
+using MixedReality.Toolkit.Core.Tests.EditMode;
+using NUnit.Framework;
+
+namespace MixedReality.Toolkit.WindowsSpeech.Tests.EditMode
+{
+    /// <summary>
+    /// This class is used to validate the package for the Mixed Reality Toolkit Windows Speech package, verifying that all
+    /// requirements are met for publishing the package to the Unity Asset Store.
+    /// </summary>
+    internal class PackageValidationTest
+    {
+        /// <summary>
+        /// Test to validate the package for the Mixed Reality Toolkit Windows Speech package, verifying that all
+        /// requirements are met for publishing the package to the Unity Asset Store.
+        /// </summary>
+        [Test]
+        public void PackageTest()
+        {
+            PackageValidatorResults results = PackageValidator.Validate("org.mixedrealitytoolkit.windowsspeech");
+            Assert.AreEqual(0, results.FailedCount, $"Failed tests found.\n{results.ToString(PackageValidatorResults.MessageType.Failed)}");
```

Comment:
Nit:  Replace `\n` with Environment.NewLine as a portability good practice.
Note: This suggestion applies to all PackageValidationTest.cs files in this PR.